cmd/daprd/main.go

```diff
@@ -330,6 +330,17 @@ func main() {
 				return eventgrid.NewAzureEventGrid(logContrib)
 			}),
 		),
+
+		// You can extend Dapr by creating your own Daprd binary by doing following
+		// 1. Create a GO project and copy this "main.go". Use Dapr and Component-contrib as library in the project
```

Comment:
Proposal:

You can extend Dapr by creating your own daprd binary. Here are the steps:
1. Create a new Go project and copy this "main.go". Use Dapr and component-contrib as libraries.
2. Uncomment the code below to add your custom components. These are exposed via gRPC endpoints.